package.json

```diff
@@ -10,7 +10,7 @@
     "build": "npm run lint && npm run compile",
     "prepublish": "npm run build",
     "lint": "eslint src",
-    "test": "echo \"To be implemented\"",
+    "test": "MONGODB_SERVICE_HOST=localhost MONGODB_DATABASE=notifications MONGODB_USER= MONGODB_PASSWORD= mocha --compilers js:babel-register -b test/spec/*.spec.js",
```

Comment:
or if you prefer you can add this package https://www.npmjs.com/package/better-npm-run that loads for you a .env file without touching source files